### PR TITLE
ilse-extensions Small modifications

### PR DESCRIPTION
**Pull request by maposatj**

### CODE REVIEW

File: search-extension/src/main/java/com/afrigis/services/search/extension/SuburbRiskTotalResponse.java [hunk 1]
```diff
@@ -11,5 +11,34 @@
...
+     * would mean the service call was a success
+     */
+    public double getTotalRisk() throws Exception;
+
+    /**
+     * AfriGIS service call response, indicates status of service call
+     *
+     * @return code
+     */
+    public int getCode();
```
Comment (demaniak) [inline search-extension/src/main/java/com/afrigis/services/search/extension/SuburbRiskTotalResponse.java:29]: @maposatj correct me if I'm wrong, but don't we inherit these methods from `Response` ?

**Review by demaniak — COMMENTED** [review]
Looks good I think, please just confirm that it is really required to add `getCode` etc to sub-interfaces of `Response`?

File: search-extension/src/main/java/com/afrigis/services/search/extension/SuburbRiskTotalResponse.java (hunk 1 above)
Comment (maposatj) [reply search-extension/src/main/java/com/afrigis/services/search/extension/SuburbRiskTotalResponse.java:29]: No @demaniak

Comment (maposatj) [reply search-extension/src/main/java/com/afrigis/services/search/extension/SuburbRiskTotalResponse.java:29]: I used these in testing responses, in the black box service lib

**Pull request merged**